config/base.json

```
@@ -29,6 +29,7 @@
 			"vault",
 			"waypoint"
 		],
+		"non_product_mobile_menu_routes": ["/", "_error"],
```

The `"_error"` item here needs to start with `/` because `router.route` outputs `"/_error"` on pages like this one: https://dev-portal-git-brkfeat-mobile-filtering-hashicorp.vercel.app/not/a/valid/route

```suggestion
		"non_product_mobile_menu_routes": ["/", "/_error"],
```